.golangci.yml

```diff
@@ -87,7 +87,7 @@ linters:
     - nakedret
     - nilerr
     - nilnil
-    - nlreturn
+    # - nlreturn
```

Comment:
In our projects we generally want to enable as many linters as possible. If there are some cases where that's not practical, we should ignore specific ones those with `//nolint:nlreturn`.